src/shared/utilities/pathUtils.ts

```diff
@@ -21,3 +21,21 @@ export function dirnameWithTrailingSlash(path: string): string {
 
     return dirname
 }
+
+/**
+ * Normalizes filepaths by lowercasing the drive letter for absolute paths on Windows. Does not affect:
+ * * relative paths
+ * * Unix paths
+ * @param filepath Filepath to normalize
+ */
+export function normalizePathIfWindows(filepath: string): string {
```

Comment:
Is this function necessary now that you are using the vscode.Uri class? Seems like we should be able to write code (in test or otherwise) that doesn't need this.

Reply:
I'll have to see how this shakes out in tests. Haven't touched those with this PR so I'll check when I implement those.